src/index.ts

```diff
@@ -122,7 +125,7 @@ export const service: KapShareService = {
       default: '{kapName}',
     },
     urlPattern: {
-      title: 'URL Pattern.',
+      title: 'URL Pattern',
       description: `URL where uploaded files can be found. Defaults to ${defaultUrlPattern}.`,
```

Comment:
My only worry with this is that `urlPattern` is only used at the end when generating the notification, while the actual URL of the file is dependent on the other fields (accountName, container etc). So if the user provides a different pattern here, the notification won't link to the right place

Reply:
It's intentional to let that be configurable. The main use case is if you have Azure blob storage fronted by your own domain/cname. For instance, I have `memes.peet.io` cnamed to my storage account, so my urlPattern is `https://memes.peet.io/{container}/{filename}`.